### PR TITLE
 Add transaction hash to replayed transaction 

### PR DESCRIPTION
**Pull request by foxfromabyss**
Extended pull request from allquantor #8546 with neccessary test support. Unfortunately the tests fail in the cli part. Would you mind helping me?

### CODE REVIEW

**Comment by foxfromabyss** [comment]
[clabot:check].

File: ethcore/src/executive.rs [hunk 1]
```diff
@@ -650,6 +650,7 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
 					exception: Some(exception),
 					gas: t.gas,
 					gas_used: t.gas,
+					hash: t.hash(),
```
Comment (sorpaas) [inline ethcore/src/executive.rs]: If `t` is fake signed this value will be invalid.

**Comment by foxfromabyss** [comment]
#8883 closes this

**Comment by 5chdn** [comment]
Okay, thanks anyways @foxfromabyss :100:

**Pull request closed without merging**